src/Microsoft.AspNetCore.Rewrite/Internal/UrlRewrite/UrlRewriteRuleBuilder.cs

```diff
@@ -53,10 +53,9 @@ public UrlRewriteRule Build()
                     _action = new RedirectAction(statusCode, url, appendQueryString);
                     break;
                 case ActionType.AbortRequest:
-                    throw new NotImplementedException("Abort Requests are not supported");
+                    throw new NotImplementedException("Abort Requests are not implemented");
```

Comment:
Is this not implemented because we couldn't or because we just didn't get around to it?